src/services/EdFi.DataManagementService.Api/EdFi.DataManagementService.Api.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <Content Include="EdFi.DataManagementService.Api.nuspec" CopyToPublishDirectory="Always" CopyToOutputDirectory="Never" />
+    <None Include="ApiSchema/DataStandard-5.0.0-ApiSchema.json" CopyToPublishDirectory="Always" CopyToOutputDirectory="Always"/>
```

Comment:
Is this the "correct" way to have the ApiSchema.json copied/bundled? Used <None> instead of <Content> just because it did what I wanted.

Reply:
Right, but we really want it bundled into a NuGet package, so we want something entirely different.